src/battle_ai_util.c

```diff
@@ -1142,7 +1142,7 @@ s32 AI_WhoStrikesFirst(u32 battlerAI, u32 battler, u32 moveConsidered)
 
 static bool32 CanEndureHit(u32 battler, u32 battlerTarget, u32 move)
 {
-    if (!BATTLER_MAX_HP(battlerTarget) || gMovesInfo[move].effect == EFFECT_MULTI_HIT)
+    if (!AtMaxHp(battlerTarget) || gMovesInfo[move].effect == EFFECT_MULTI_HIT)
```

Comment:
Do we have both `AtMaxHp` and `IsBattlerAtMaxHp` or is this a typo, and if it's not a typo, why do we have both?

Reply:
the other one is used for ai calcs

Reply:
Gotcha, so CanEndureHit was actually using the wrong one of these up to this point, just to clarify?

Will approve and merge if that's the case. I think it might be worth considering some clarifying naming convention for AI stuff vs non-AI stuff, as `AtMaxHp` doesn't inherently carry any "this is for the AI" meaning if that makes sense. Not for this PR at all obviously :)

Reply:
Would it be okay to add the prefix `Is` to `BattlerTurnDamaged` so `IsBattlerTurnDamaged`?

Reply:
I would actively prefer that yeah :D

Reply:
>I wouldn't say a wrong one. It basically checks the same thing.
>I add an AI_ prefix to clarify it
Can we add an AI prefix to this one either in this or a subsequent PR then?

Reply:
addressed both points

Reply:
Perfect, will approve after CI :)